first-snap/python/build.yaml

```diff
@@ -0,0 +1,50 @@
+linux:
+  auto:
```

Comment:
May be worth clarifying when @therealjuan is back, but if I recall from a meeting with advocacy we decided to have only one way of building by default (was it auto?) - I think it was bound to some feature being released for snapcraft or snapd or core for virtualisation.

Not sure about the details, but I think there was meant to be only one way to build (no docker instructions I believe). Also something worth clarifying - not sure if we need different instructions per language/tool - or is it just one per OS - basically 'how to run snapcraft on given OS'.